### ISSUE TITLE
IMOS - check_dimension

### ISSUE THREAD

**Issue by ggalibert** [aodn/compliance-checker#105]
I am getting this result:

```
var                                    :3:   161/162 :  
    data_variable                      :3:    14/15 :  
        GDOP                           :3:     1/ 2 :  
            check_dimension            :3:     0/ 1 : data variable should have
                                                      at least one of the
                                                      dimensions TIME, LATITUDE,
                                                      LONGITUDE, DEPTH
```

for this [kind of file](http://thredds.aodn.org.au/thredds/catalog/IMOS/eMII/demos/ACORN/monthly_gridded_1h-avg-current-map_non-QC/TURQ/2012/catalog.html?dataset=IMOS/eMII/demos/ACORN/monthly_gridded_1h-avg-current-map_non-QC/TURQ/2012/IMOS_ACORN_V_20121001T000000Z_TURQ_FV00_monthly-1-hour-avg_END-20121029T180000Z_C-20121030T160000Z.nc.gz) (CODAR ACORN radar):

```
dimensions:
        TIME = UNLIMITED ; // (1 currently)
        I = 60 ;
        J = 59 ;
...
        double LATITUDE(I, J) ;
                LATITUDE:standard_name = "latitude" ;
                LATITUDE:long_name = "latitude" ;
                LATITUDE:units = "degrees_north" ;
                LATITUDE:axis = "Y" ;
                LATITUDE:valid_min = -90. ;
                LATITUDE:valid_max = 90. ;
                LATITUDE:reference_datum = "geographical coordinates, WGS84 datum" ;
        double LONGITUDE(I, J) ;
                LONGITUDE:standard_name = "longitude" ;
                LONGITUDE:long_name = "longitude" ;
                LONGITUDE:units = "degrees_east" ;
                LONGITUDE:axis = "X" ;
                LONGITUDE:valid_min = -180. ;
                LONGITUDE:valid_max = 180. ;
                LONGITUDE:reference_datum = "geographical coordinates, WGS84 datum" ;
        float GDOP(I, J) ;
                GDOP:_FillValue = 999999.f ;
                GDOP:long_name = "radar beam intersection angle" ;
                GDOP:units = "Degrees" ;
                GDOP:valid_min = 0.f ;
                GDOP:valid_max = 180.f ;
                GDOP:coordinates = "LATITUDE LONGITUDE" ;
                GDOP:comment = "This angle is used to assess the impact of Geometric Dilution of Precision. If angle >= 150 or <= 30, then QC flag will not be lower than 4 (see abstract)." ;
...
```

I'm wondering about the legitimacy of such a test.

**Comment by mhidas** [comment]
@ggalibert You're right, in that case it doesn't make sense. This is not actually from the IMOS conventions doc. I thought it was a reasonable check that can later be replaced by more featureType-specific dimension checks. Obviously I hadn't considered all sampling geometries. Probably easiest to just remove this check.

**Comment by mhidas** [comment]
The above linked file is irrelevent. [Here](http://thredds.aodn.org.au/thredds/catalog/IMOS/ACORN/gridded_1h-avg-current-map_non-QC/TURQ/2016/06/06/catalog.html?dataset=IMOS/ACORN/gridded_1h-avg-current-map_non-QC/TURQ/2016/06/06/IMOS_ACORN_V_20160606T010000Z_TURQ_FV00_1-hour-avg.nc) is one that shows this issue.

**Comment by mhidas** [comment]
@ggalibert See #106

**Comment by mhidas** [comment]
Closed via #106
